src/schemas/index.js

```diff
@@ -0,0 +1,28 @@
+var apollo = require('apollo-server-express');
```

Comment:
Avoid `var`. use `const` or `let` everywhere in your code. As a rule of thumb you could consider, always use `const` unless you're mutating the variable, then `let`.

Reply:
OK, I think I understand it now.